scripts/generate-cov-report.sh

```diff
@@ -11,27 +15,28 @@ LCOV=0
 
 cleanup() {
     find . -name "*profraw" -exec rm {} \;
-    rm json5format.profdata
+    if [[ -d "./target/coverage" ]]; then rm -rf ./target/coverage; fi
 }
 
 generate_profile() {
-    RUSTFLAGS="-C instrument-coverage" cargo test --workspace -- --test-threads=1
+    cargo test --workspace -- --test-threads=1
 }
 
 merge_report() {
+    mkdir -p ./target/coverage/
     local FILES=$(find . -name "*profraw")
-    llvm-profdata merge -sparse $FILES -o json5format.profdata
+    llvm-profdata merge -sparse $FILES -o ./target/coverage/json5format.profdata
 }
 
 generate() {
     cleanup
     generate_profile
-    merge_report
+    # merge_report
```

Comment:
I'm not quite sure what this does and 'show_overview' and 'show_report' did not work on my local install. Can we remove those?

Reply:
Can you please replace the `FILES` definition in `show_overview` with

```sh
local FILES=$(find ./target/debug/deps/*tests* -type f -executable)
```

This should repair it. It provides you a nice overview of all files and their code coverage. Not yet perfect but a nice terminal overview.

Reply:
Oh and in `show_report`, please add

```sh
local FILES=$(find ./target/debug/deps/ -type f -executable)
```

at the top and replace the last `CMD` assignment after the for loop with this

```sh
CMD="$CMD --show-instantiation-summary --Xdemangler=rustfilt | less -R"
```